portal/static/js/admin.js

```diff
@@ -562,7 +562,7 @@
                 $("#patientReportModal").modal("show");
                 this.patientReports.loading = true;
                 var self = this, tnthDates = self.getDependency("tnthDates"), tnthAjax = self.getDependency("tnthAjax");
-                tnthAjax.patientReport(patientId, function(data) {
+                tnthAjax.patientReport(patientId, false, function(data) {
```

Comment:
It might be easier to extend your functions if you use an `options` object to specify additional (optional) arguments. 

You wouldn't need to rewrite each function call every time you add a argument to the definition, and it would be more clear what options you're passing when you call your function.

...looks like you did that with some other functions in this PR, nice! :)